meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+galaxy_info:
+  author: Jiri Tyr
+  description: Ansible role creates RPM build environment
+  license: MIT
+  min_ansible_version: 1.9
+  platforms:
+    - name: EL
+      versions:
+        - all
+    - name: Fedora
+      versions:
+        - all
+  galaxy_tags: []
+dependencies:
+  - jtyr.config_encoder_filters
```

Comment:
Please change this like to:

```
- config_encoder_filters
```

and follow [installation instructions](https://github.com/jtyr/ansible-config_encoder_filters#installation) to install the dependency.